test/properties.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+var jsc = require('jsverify');
+
+exports.idempotent = function(f) {
+  return jsc.checkForall(useful, function(x) {
+    return f(x) === f(f(x));
```

Comment:
I don't think so. `x` is not necessarily a primitive value, presumably, so:

```javascript
> const f = R.sortBy(S.I)
> const x = ['c', 'a', 'b']
> f(x)
['a', 'b', 'c']
> f(f(x))
['a', 'b', 'c']
> f(x) === f(f(x))
false
```

Let's use `R.equals` here.